src/styles/homestyle/style.css

```diff
@@ -0,0 +1,375 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Oxygen:wght@300;400;700&display=swap');
+* {box-sizing: border-box;}
+
+body { 
+  margin: 0;
+ font-family: 'Oxygen', sans-serif;
+  overflow-x: hidden;
+}
+/*header*/
+.header {
+  overflow: hidden;
+  background-color: #fff;
+  padding: 20px 10px;
+  text-decoration: none;
+  padding: 20;
+    position: fixed;
+    width: 100%;
+    left: 0;
+    right: 50;
+    -webkit-box-shadow: 0 7px 13px 0 rgba(0, 0, 0, 0.1);
+    box-shadow: 0 7px 13px 0 rgba(0, 0, 0, 0.1);
+    top: 0;
+    z-index: 999;
+    padding: 15px 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)